Dockerfile

```
@@ -220,7 +220,8 @@ ENV PATH $PATH:/home/docs/.conda/bin
 RUN rm -f Miniconda2-${RTD_CONDA_VERSION}-Linux-x86_64.sh
 
 # Install Rust
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
+ENV RUST_VERSION 1.46.0
```

```suggestion
ENV RTD_RUST_VERSION 1.46.0
```

I'm adding the prefix here since we had some issues with commands reading similar variables and getting confused. After that, we start adding `RTD_` to all our variables.

---

Good call. Done!